project/leonhsi/README.md

```diff
@@ -0,0 +1,42 @@
+# Bounding Volume Hierarchy Trees Generator
+
+## Basic Information
+
+Bounding Volume Hierarchies (BVH) trees are an acceleration sturcture for ray tracing algorithm. The problem of ray tracing is to find the nearest intersection with a given ray and scene. In practice, it has to evaluate billions of such operations as the scene consists of millions of primitives, thus scene primitives are commonly arranged in spatial data structures to accelerate the search. BVH are such data structure that used mostly nowadays.
```

Comment:
Do you plan to make your implementation for millions of primitives?  Please make it clear the scale of problems that you would like to deal with somewhere in the proposal.